### PR TITLE
PLAT-1992: Create zen-sidebar-nav component

### PR DESCRIPTION
**Pull request by chavlji**
[JIRA ticket](https://reciprocitylabs.atlassian.net/browse/PLAT-1992)
Based on https://github.com/reciprocity/zen-ui/pull/332

### CODE REVIEW

**Review by f-jauregui — APPROVED** [review]
👌

File: src/components/zen-sidebar-nav/zen-sidebar-nav.tsx [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+      <Host>
+        <ZenSidebar class="sidebar" collapsed-size="32" expanded={this.expanded}>
+          <slot></slot>
+
+          <ZenSpace padding="lg md" class="footer" vertical>
+            <a href="https://placeholder.com/" target="_blank">
+              Privacy Policy
+            </a>
+            <div>Copyright © 2021</div>
+          </ZenSpace>
```
Comment (homer0) [inline src/components/zen-sidebar-nav/zen-sidebar-nav.tsx:35]: This should be the default value of a named slot.

File: src/components/zen-sidebar-nav/zen-sidebar-nav.tsx [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+          <ZenIcon
+            slot="wrapChildren"
+            role="button"
+            padding="md"
+            size="md"
+            class="collapse-icon hover-ignore"
+            icon={faChevronDoubleLeft}
+            onClick={() => this.toggle()}
+          ></ZenIcon>
```
Comment (homer0) [inline src/components/zen-sidebar-nav/zen-sidebar-nav.tsx:45]: This should be toggable using a prop, as the first version won't include it.

**Review by homer0 — APPROVED** [review]
For the last PRs

**Pull request merged**